integration_test/README.md

```diff
@@ -248,7 +248,7 @@ a hop. The following is sorted roughly in descending order of usefulness.
 ├── logs
 |   ├── build_and_test.txt
 |   ├── sponge_log.xml
-|   └── TestThirdPartyApps_ops-agent_debian-10_nginx
+|   └── TestThirdPartyApps_ops-agent_debian-12_nginx
```

Comment:
The packages below still say `debian10`…

Reply:
[e18feca](https://github.com/GoogleCloudPlatform/ops-agent/pull/1752/commits/e18feca17e427d43c5d2ed801276a535b2105733)